error_default.go

```diff
@@ -64,6 +64,14 @@ type DefaultError struct {
 	// required: true
 	ErrorField string `json:"message"`
 
+	// Temporary marks an error as temporary.
+	//
+	// Temporary errors are usually caused by a temporary issue and should be
+	// retried by the client after a reasonable backoff.
+	//
+	// required: true
+	TemporaryField bool `json:"temporary"`
```

Comment:
I'd inverse this and say "no_retry" because this error might not even make it to the caller if something else fails (e.g. internal cluster network error).

Reply:
No retry feels very weird, as you have to say `NoRetry = false` to specify it is retryable. But you want the default to be retry right?

Reply:
Yes, default should be retry for most 5xx errors (e.g. 502 bad gateway) and and some 4xx errors (e.g. 429 too many requests).

Usually, one would use the `Retry-After` to indicate when the retry can happen again. However, most of the infrastructure currently doesn't support setting this value correctly.

To disable retry explicitly, one could set `Retry-After` to a date far in the future, for example.

In summary, I don't think setting this as a JSON key is an appropriate fix as it is very application specific. We should probably work with default HTTP headers that are understood by most clients already.

For the console, we can narrow down the codes that need retry, e.g. 429, 503, 502, 500